lib/article_json/export/common/html/elements/embed.rb

```diff
@@ -20,7 +20,8 @@ def export
             private
 
             def embed_node
-              create_element(:div, class: 'embed') do |div|
+              type = @element.embed_type || :type
```

Comment:
What is the `|| :type` statement meant to do? If there is no `embed_type` defined, it would be okay to not have a class for it, wouldn't it? (also, I think if there's no `embed_type` defined, the code would explode anyways at some other point..)

Reply:
ouch, this is a leftover from a debugging session, removing it.